src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+export default function ArticlePage() {
+    const [article, setArticle] = useState({});
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
This isLoading state is declared and updated but not used - there is a very good reason to use it as you have done  in your ArticleList component, please do the same here.

Reply:
Fixed